ingress-auth.yaml

```diff
@@ -0,0 +1,19 @@
+v1:
+  requires:
+    type: object
+    properties:
+      service:
+        type: string
+      port:
+        type: int
```

Comment:
I guess, `integer` and `int` are the same here, but just for sake of consistency lets just use one of the 2.